setup/install_gatling/README.md

```diff
@@ -0,0 +1,68 @@
+<!-- TODO: This should replace the main readme.md file once we're all settled -->
```

Comment:
you could, for now, just add a link to this from the main readme

Reply:
Good call